src/search-scenario/azext_search_scenario/azext_metadata.json

```diff
@@ -0,0 +1,4 @@
+{
+    "azext.isPreview": true,
+    "azext.minCliCoreVersion": "2.20.0"
```

Comment:
Do we have any dependency of minimum cli core version?

Reply:
No other dependencies except `azure.cli.core`. Since it uses the color framework, so the `minCliCoreVersion` should be `2.20.0` as `az next`